src/repository/logic.py

```diff
@@ -382,10 +386,11 @@ def get_preprint_if_id(preprint_id):
 
 # TODO: Update this
 @cache(300)
-def list_articles_without_subjects():
+def list_articles_without_subjects(repository):
```

Comment:
Should this be an optional argument as a kwarg that defaults to `None`?

Reply:
👍🏻